Qapple/Qapple/SourceCode/Feature/0.Root/RootFeature.swift

```diff
@@ -14,19 +14,34 @@ struct RootFeature {
     struct State: Equatable {
         var path = StackState<Path.State>()
         var questionTab = QuestionTabFeature.State()
+        var bulletinBoardTab = BulletinBoardFeature.State()
     }
     
     enum Action {
         case path(StackActionOf<Path>)
         case questionTab(QuestionTabFeature.Action)
+        case bulletinBoardTab(BulletinBoardFeature.Action)
     }
     
     var body: some ReducerOf<Self> {
         Scope(state: \.questionTab, action: \.questionTab) {
             QuestionTabFeature()
         }
+        Scope(state: \.bulletinBoardTab, action: \.bulletinBoardTab) {
+            BulletinBoardFeature()
+        }
         Reduce { state, action in
             switch action {
+                
+            case let .path(action):
+                switch action {
+                default: return .none
+                }
+                
+            case let .bulletinBoardTab(.boardButtonTapped(board)):
+                state.path.append(.commentView(.init(post: board)))
+                return .none
```

Comment:
게시글을 눌렀을 때 commentView로 이동하는 네비게이션을 적용했는데 위 영상처럼 빈 화면만 나오는 데 이유를 알 수 있을까요??
```suggestion
                    .onTapGesture {
                        if !board.isReported {
                            store.send(.boardButtonTapped(board))
                        } else {
                            HapticService.notification(type: .warning)
                            store.send(.reportButtonTapped)
                        }
                    }
```
여기서 cell을 눌렀을 때 호출됩니다!

Reply:
그 이유는 하단 코드 이미지의 Destination에 어떠한 View도 정의가 되어있지 않아서 그렇습니다!
![image](https://github.com/user-attachments/assets/cf03c42d-e46f-4cb5-ab99-de14334f6bd0)

아래와 같이 어디로 이동해야하는지 View를 정의해주셔야 합니다!
~~~swift
switch store.case {
case let .bulletinBoardView(store): BulletinBoardView(store: store)
case let .commentView(store): CommentView(store: store)
}
~~~

Reply:
아하 다음에 적용해봐야겠어요!!